test_goto.py

```diff
@@ -71,63 +71,87 @@ def func():
+    @with_goto
+    def func():
+        for i in range(2):
+            for j in range(2):
+                for k in range(2):
                     goto .end
-            label .end
-            return (i, j)
+        label .end
+        return (i, j, k)
 
-        assert func() == (0, 0)
+    assert func() == (0, 0, 0)
 
-    def test_jump_out_of_nested_3_loops():
-        def func():
-            for i in range(2):
-                for j in range(2):
-                    for k in range(2):
+def test_jump_out_of_nested_4_loops():
+    @with_goto
+    def func():
+        for i in range(2):
+            for j in range(2):
+                for k in range(2):
+                    for m in range(2):
                         goto .end
-            label .end
-            return (i, j, k)
-
-        pytest.raises(SyntaxError, with_goto, func)
-else:
-    def test_jump_out_of_nested_4_loops():
-        @with_goto
-        def func():
-            for i in range(2):
-                for j in range(2):
-                    for k in range(2):
-                        for m in range(2):
-                            goto .end
-            label .end
-            return (i, j, k, m)
+        label .end
+        return (i, j, k, m)
 
-        assert func() == (0, 0, 0, 0)
+    assert func() == (0, 0, 0, 0)
 
-    def test_jump_out_of_nested_5_loops():
-        def func():
-            for i in range(2):
-                for j in range(2):
-                    for k in range(2):
-                        for m in range(2):
-                            for n in range(2):
-                                goto .end
-            label .end
-            return (i, j, k, m, n)
+def test_jump_out_of_nested_5_loops():
+    @with_goto
+    def func():
+        for i in range(2):
+            for j in range(2):
+                for k in range(2):
+                    for m in range(2):
+                        for n in range(2):
+                            goto .end
+        label .end
+        return (i, j, k, m, n)
 
-        pytest.raises(SyntaxError, with_goto, func)
+    assert func() == (0, 0, 0, 0, 0)
+
+def test_jump_out_of_nested_11_loops():
+    @with_goto
+    def func():
+        x = 1
+        for i1 in range(2):
+            for i2 in range(2):
+                for i3 in range(2):
+                    for i4 in range(2):
+                        for i5 in range(2):
+                            for i6 in range(2):
+                                for i7 in range(2):
+                                    for i8 in range(2):
+                                        for i9 in range(2):
+                                            for i10 in range(2):
+                                                for i11 in range(2):
+                                                    # These are more than 256 bytes of bytecode
+                                                    x += x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x+x
```

Comment:
Any chance you can wrap this to not exceed 79 characters per line?

Reply:
Okay, I understand you don't want >79 characters lines at all and will update my latest pull request accordingly.

Reply:
Updated this pull request accordingly as well (and also removed the trailing whitespace that my editor loves to add)